android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.junit5 = '5.3.2'
     ext.assertj_version = '3.13.2'
     ext.mockk_version = '1.10.0'
-    ext.gradle_maven_publish = '0.15.1'
+    ext.gradle_maven_publish = '0.18.0'
```

Comment:
Had to bump this one to get the env variables working. 
This library is very liberal with its version numbers, it looks like they remove stuff on minor releases often. Didn't find any usage coming from us for removed stuff though.